src/game.h

```diff
@@ -572,6 +572,10 @@ class game
         int assign_npc_id();
         Creature *is_hostile_nearby();
         Creature *is_hostile_very_close();
+        bool is_map_cache_valid = false;
+        inline void invalidate_map_cache() {
```

Comment:
There's no need to explicitly `inline`: everything defined in the header will be inlined anyways (even in cpp, it's a single line assignment of a primitive data type, which is a perfect candidate for inlining).